ocs_ci/templates/ocs-deployment/storageclass_thin-csi-odf.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "false"
+  name: thin-csi-odf
+parameters:
+  StoragePolicyName: "vSAN Default Storage Policy"
+provisioner: csi.vsphere.vmware.com
+allowVolumeExpansion: true
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
Instead of maintaining the yaml why now get the output of default sc i.e `thin-csi` and utilize that just by changing the name and storage policy name?

Reply:
we can do that way as well. Apart from changing name and storage policy name, we need to chnage annotations, remove uid, creationtimestamp and resourceversion as well.

So maintaining separate yaml for deployment is easier and error free.